assets/stylesheets/layout/_main.scss

```diff
@@ -583,3 +592,24 @@ body.newdash div.wordpress-com-extension-promo {
 html.no-scroll {
 	overflow: hidden;
 }
+
+.is-section-sites #primary,
```

Comment:
Ideally I think a component should be responsible for choosing whether it scrolls or not. Here we're making the assumption that most everything in `#primary` does want to scroll.

The `Main` component seems like a good candidate that enables scrolling because most components use it for the content that ends up being scrollable.